### ISSUE TITLE
Add support for OPUS files.

### ISSUE THREAD

**Issue by zayuim** [gateship-one/odyssey#12]
Playback of the [xiph.org opus audio format](https://en.wikipedia.org/wiki/Opus_%28audio_format%29) is currently not supported (e.g. they current don't show up in the library). I would like to see support added.

**Comment by djselbeck** [comment]
Android 5 should support opus in matroska container format which should be supported by odyssey.
If they don't show up the android mediascanner probably did not parse them correctly. Are they perhaps in the ogg container format (not sure if android supports them)

I will investigate this.

**Comment by gnome17** [comment]
It seemed that the android mediascanner did not parse .opus files. We will add support for those files in the next version of odyssey. Probably next week. 
Please notice that with this fix you will only be able to play those files from the odyssey file browser.

**Comment by zayuim** [comment]
Currently all my opus files are in the .opus container.

It kind of sucks that you can't implement them into the library as well. But I guess that's the fault of the default mediascanner, right?

**Comment by gnome17** [comment]
We investigated this issue a bit more and found out that Android (tested with version 6) does indeed support the playback of .opus files. Android 5.0 does not!

Sadly Android 6 still does not parse the files for the media library but we added a workaround, that allows to play .opus files from the file browser which tries to extract the Metadata when adding the files to the play queue. This works for our test devices.
